src/mmrelay/meshtastic_utils.py

```
@@ -900,32 +1133,100 @@ def _schedule_ble_future_cleanup(
+    """
+
+    def _dispose(done_future: Future[Any]) -> None:
+        if done_future.cancelled():
+            return
+
+        late_iface = fallback_iface
+        try:
+            result = done_future.result()
+            if result is not None:
+                late_iface = result
+        except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)